lib/request.js

```diff
@@ -37,7 +37,7 @@ function makeRequest(options, cb) {
             });
             res.on('end', () => {
                 if (cb) {
-                    const body = options.keepAsBuffer ? Buffer.concat(data) : Buffer.concat(data).toString();
```

Comment:
no need for the `keepAsBuffer` option anymore?
how come?
if it's not needed, remove it from calls that use it.